packages/sources/css/src/components/text-input/src/index.css

```diff
@@ -3,6 +3,12 @@
 @import '@vtmn/css-design-tokens/src/spacings';
 @import '@vtmn/css-design-tokens/src/typography';
 
+.vtmn-text-input_label,
+.vtmn-text-input,
+.vtmn-text-input_helper-text {
+  font-family: 'Roboto Condensed', 'system-ui', '-apple-system', sans-serif;
```

Comment:
```suggestion
  font-family: var(--vtmn-typo_font-family);
```

Reply:
Added the CSS Custom Properties provided by our design tokens package to add Roboto.
For your information, Roboto Condensed is not the font that we have chosen for the text input component (due to lisibility).
Another information, following our survey #4, the font-family of the button will also change in the future from Roboto Condensed to Roboto.